analysed-packages/tcpdump/version-4.99.1/tcpdump-4.99.1-OSS-disclosure.txt

```diff
@@ -1239,6 +1229,13 @@ FOR A PARTICULAR PURPOSE.
 
 Dual-license
 
+License LicenseRef-BSD-4-Clause-Shortened-e96ddab5f8fffe47de17b5812d71176a AND
```

Comment:
Can you please change this to:
License: GPL-1.0-or-later OR Permission-Notice
The BSD-4-Clause is not to be considered for the dual licensing, nevertheless the overall license of the file is 
BSD-4-Clause-Shortened-e96ddab5f8fffe47de17b5812d71176a AND (GPL-1.0-or-later OR Permission-Notice)